### PR TITLE
Fix/styling issues new pages

### PR DESCRIPTION
**Pull request by AdityaKhatri**
- Addresses
    * https://zube.io/deep/deep/c/3723
    * https://zube.io/deep/deep/c/3719
    * https://zube.io/deep/deep/c/3720
    * https://zube.io/deep/deep/c/3721
    * https://zube.io/deep/deep/c/3725
    * https://zube.io/deep/deep/c/3725

## Changes

* Fix above mentioned issues

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

## This PR contains valid:

- [x] permission checks
- [x] translations

### CODE REVIEW

File: src/newViews/ProjectEdit/ProjectDetailsForm/index.tsx [hunk 1]
```diff
@@ -441,6 +547,10 @@ function ProjectDetailsForm(props: Props) {
                     </Button>
                 )}
             />
+            <Prompt
+                when={!pristine}
+                message={_ts('common', 'youHaveUnsavedChanges')}
+            />
```
Comment (AdityaKhatri) [inline src/newViews/ProjectEdit/ProjectDetailsForm/index.tsx:553]: @subinasr , this much is okay for Prompt.

Comment (subinasr) [reply src/newViews/ProjectEdit/ProjectDetailsForm/index.tsx:553]: Yeah, thanks bro!

**Review by samshara — APPROVED** [review]

**Comment by subinasr** [comment]
The issue in https://zube.io/deep/deep/c/3719 (no. 3) is not solved. The scroll bar appears in the stakeholder section even when there are no organizations added.

Also, I think the same issue has been listed twice in the 'Addresses' section :P

**Comment by subinasr** [comment]
image
Here. This scroll bar should disappear when there are no items, shouldn't it?

@subinasr , this is from deep-ui and I've created another issue for it: https://github.com/the-deep/client/issues/1923

**Review by subinasr — APPROVED** [review]

**Review by tnagorra — APPROVED** [review]

**Pull request merged**